src/components/TP6mAggregate.jsx

```diff
@@ -231,31 +229,31 @@ class TP6mAggregate_ extends Component {
               .where({ test })
               .along('platform')
               .enumerate()
-              .map((row, i) => {
+              .map(row => {
```

Comment:
Could we get a short explanation of where this `i` (and, presumably, the `index` below) went, and why they're no longer necessary?  This is the type of "why?" question that I'd love to see in a commit message.

Reply:
https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/415/commits/c25330bce8072620f336b8398a96c7ca1b7e9669 is the commit, and description, of what's happening to these index variables.